ssz_serialization/bitseqs.nim

```diff
@@ -85,6 +85,7 @@ template loopOverWords(lhs, rhs: BitSeq,
   const hasRhs = astToStr(lhs) != astToStr(rhs)
 
   let bytesCount = len Bytes(lhs)
+  doAssert bytesCount > 0 # Ensure that `init` was called on the `BitSeq`.
```

Comment:
Perhaps you can add that comment in the doAssert to improve diagnostics a little more ;)

Reply:
Thanks. I changed it to the comment. When testing, the comment was not displayed though.